index.html

```diff
@@ -100,174 +116,199 @@ <h2 class="title">Software Developer</h2>
       </ul>
     </div>
 
-    <section class="work-section">
+    <section>
       <h3 class="work-header">My recent work</h3>
```

Comment:
## OPTIONAL
 
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![anu](https://user-images.githubusercontent.com/69638013/160129304-7f5b8df0-ebb1-4a37-b762-1bdbfed7e1dc.png)  | ![Screenshot_1](https://user-images.githubusercontent.com/69638013/160129477-c25c82f8-4dc3-4796-852e-4ebb771a7328.png)


- [ ] Kindly align the header text on the mobile to look like the figma design
- [ ] Kindly make  all the  horizontal lines on  your project look like the figma design color 👍🏿